frontend/src/lib/components/PropertyKeyInfo.js

```diff
@@ -173,6 +173,52 @@ export const keyMapping = {
             description: 'The initial type of device that was used (first-touch).',
             examples: ['Mobile', 'Tablet', 'Desktop'],
         },
+        $pageview: {
+            label: 'Pageview',
+            description: 'When a user loads (or reloads) a page.',
+        },
+        $pageleave: {
+            label: 'Pageleave',
+            description: 'When a user leaves a page.',
+        },
+        $autocapture: {
+            label: 'Autocapture',
```

Comment:
What are your thoughts on whether 'UI Interaction' would be more accurate for this one? Since most (all?) of these I believe are auto captured.

Reply:
Do you mean to change the label of this to `UI Interaction` here?

Reply:
For the label name specifically?

Reply:
Yea for the label name

Reply:
hmm would that provide more clarity than what the description for it already delivers?

Reply:
Discussed offline– going to keep it as is so we're consistent roughly keeping the name consistent with the actual field.